### DISCUSSION TITLE
Russian language support?

### DISCUSSION THREAD

**Discussion by zolotaya-ruda** [hecrj/iced#1621] [Q&A]
Is supported iced russian language? If so, how can I enable this? Because i see it when try enter russian text. My platform is Kali.

![photo_2022-04-29_18-37-17](https://user-images.githubusercontent.com/75062897/209473075-51450941-8dfc-48c9-8ff9-5039957d863c.jpg)

**Comment by 13r0ck** [comment]
Not yet. Text rendering is currently broken in iced. But https://github.com/pop-os/cosmic-text is nearing completion, then once used by iced this will be fixed. Though that may take a few months.

**Comment by ids1024** [comment]
You may have better luck if you build with the `default_system_font ` feature. That should work better as long as the system default font has Cyrillic characters.

But yeah, proper support for text shaping, font fallbacks, etc. has not been integrated quite yet, so some things won't be right.
